mesonbuild/modules/i18n.py

```diff
@@ -62,6 +62,11 @@ def merge_file(self, state, args, kwargs):
 
         kwargs['command'] = ['msgfmt', '--' + file_type,
                              '--template', '@INPUT@', '-d', podir, '-o', '@OUTPUT@']
+
+        extra_args = mesonlib.stringlistify(kwargs.pop('args', []))
+        if extra_args is not None:
```

Comment:
or just `if extra_args`. The `None` check is certainly wrong since it would be `[]`.